package.json

```diff
@@ -28,6 +28,7 @@
     "active-model-adapter": "1.13.6",
     "broccoli-asset-rev": "^2.1.2",
     "broccoli-static-compiler": "^0.2.1",
+    "broccoli-unwatched-tree": "^0.1.1",
```

Comment:
I'm getting an error using your fork on a new app, I think this needs to be in `dependencies`